cou/utils/openstack.py

```diff
@@ -330,8 +352,8 @@ def find_compatible_versions(cls, charm: str, version: str) -> list[OpenStackRel
         return []
 
     @classmethod
-    def is_charm_supported(cls, charm: str) -> bool:
-        """Check if a charm is supported or not in the OpenStackCodenameLookup.
+    def is_primary_charm_supported(cls, charm: str) -> bool:
```

Comment:
maybe `is_charm_version_supported` represent better what we are trying to do because we support a group of charms, but some of then we don't support version, right?

Reply:
ok